### ISSUE TITLE
Why Info.plist on the podspec ?

### ISSUE THREAD

**Issue by magnumrocha** [noheltcj/multiplatform-kotlin-hackathon-starter#3]
The command `pod lib  lint --verbose` don't works because this line:

https://github.com/noheltcj/multiplatform-kotlin-hackathon-starter/blob/fd23ee2583a0dc5d756769f5eccf4d07efbac66c/StarterCore.podspec#L13

My setup:
```
$ xcodebuild -version
Xcode 10.2
Build version 10E125

$ pod --version
1.6.1
```

When I remove or comment this line, the command executes successful

**Comment by noheltcj** [comment]
There's no need for that line to exist. Your PR #4 will take care of it.

**Comment by noheltcj** [comment]
Resolved by #4
